hr_timesheet_task_required/README.rst

```diff
@@ -33,6 +33,15 @@ mandatory field.
 .. contents::
    :local:
 
+Configuration
+=============
+
+To make task selection mandatory on project's timesheets:
+
+# Go to *Project > Projects* and edit the project
+# Enable timesheets by checking *Allow timesheets*
+# Make task selection mandatory by checking *Require Tasks on Timesheets*
```

Comment:
This is missing to mention that the default value for this field can be set at company level.

Reply:
Added